contracts/src/v0.8/dev/ArbitrumSequencerUptimeFeed.sol

```diff
@@ -117,11 +117,12 @@ contract ArbitrumSequencerUptimeFeed is
    * @notice versions:
    *
    * - ArbitrumSequencerUptimeFeed 1.0.0: initial release
+   * - ArbitrumSequencerUptimeFeed 1.1.0: change `FLAG_L2_SEQ_OFFLINE` to current Arbitrum seq offline flag
    *
    * @inheritdoc TypeAndVersionInterface
    */
   function typeAndVersion() external pure virtual override returns (string memory) {
-    return "ArbitrumSequencerUptimeFeed 1.0.0";
+    return "ArbitrumSequencerUptimeFeed 1.1.0";
```

Comment:
nit: as the contract wasn't released yet we don't need to bump, but I'm ok either way.

Reply:
Makes sense. I dropped the commit so we won't bump :)

Reply:
cc @yosriady

Reply:
Noted